images/win/scripts/Tests/AzModules.Tests.ps1

```diff
@@ -0,0 +1,46 @@
+# Get modules content from toolset
+$modules = (Get-ToolsetContent).azureModules
+
+$modulesRootPath = $env:PSMODULES_ROOT_FOLDER
+
+foreach ($module in $modules) {
+    Describe "$($module.name)" {
+
+        if ($module.default) {
+            Context "default" {
+                $moduleInfo = @{ moduleName = $module.name; moduleDefault = $module.default }
+                It "$($module.name) version $($module.default) is default" -TestCases $moduleInfo {
+                        $moduleVersion = (Get-Module -ListAvailable -Name $moduleName).Version.ToString()
```

Comment:
Why do we need `-ListAvailable` here? Looks like you don't use it below

Reply:
Because we don't invoke `Import-Module` here as it's done below, so we need to list all available modules